pkgs/development/python-modules/jaxlib/bin.nix

```diff
@@ -18,12 +18,14 @@
 , autoPatchelfHook
 , buildPythonPackage
 , config
-, cudnn ? cudaPackages.cudnn
```

Comment:
is cudnn no longer a dependency?

Reply:
Wasn't this line redundant with this snippet ?
```nix
let
  inherit (cudaPackages) cudatoolkit cudnn;
in
```